git/foundations_git/introduction_to_git.md

```diff
@@ -8,9 +8,9 @@ Git is like a really **epic save button** for your files and directories. Offici
 
 However, a *save* in Git records differences in the files and folders AND keeps a **historical record of each save**. This feature is a game changer. As an individual developer, Git enables you to review how your project grows and to easily look at or restore file states from the past. Once connected to a network, Git allows you to push your project to GitHub or other alternatives such as: Bitbucket, Beanstalk, or GitLab for sharing and collaborating with other developers.
 
-Please note, we **only** support GitHub within our curriculum, and will not help troubleshoot the alternatives.
+Make sure not to confuse Git and GitHub. <span id="git-local"></span>While Git works on your *local* machine, <span id="github-remote"></span>GitHub is a *remote* storage facility on the web for all your coding projects. This means that by learning Git, you will get to showcase your portfolio and collaborate with other programmers on GitHub! This is really important because almost all software development companies consider using Git to be an **essential skill** for modern web developers. Having a GitHub portfolio will provide proof to future potential employers as to what you are capable of.
```

Comment:
```suggestion
Make sure not to confuse Git and GitHub. <span id="git-local"></span>While Git works on your **local** machine, <span id="github-remote"></span>GitHub is a **remote** storage facility on the web for all your coding projects. This means that by learning Git, you will get to showcase your portfolio and collaborate with other programmers on GitHub! This is really important because almost all software development companies consider using Git to be an essential skill for modern web developers. Having a GitHub portfolio will provide proof to future potential employers as to what you are capable of.
```

I am of the opinion there hardly ever is a need for bold or cursive text. In this case I think it's good to emphasise local and remote however, and I'd suggest making them bold instead.